3506/3/app/helpers/restaurant_helper.rb

```diff
@@ -0,0 +1,13 @@
+#:reek:UtilityFunction
+module RestaurantHelper
+  def count_average_rating(restaurant_id)
+    Review.where(restaurant_id: restaurant_id).average(:rating).round(2)
+  end
+
+  def make_review
+    @user = User.find(session[:user_id])
```

Comment:
If you put `current_user` to ApplicationController it will be current_user, so this line will unnecessary